Controls/Recipient.c

```diff
@@ -4,19 +4,20 @@
  * The recipient will read mail from the mailbox.
  */
 
+
+/**
+ * @brief Print the mail in mailbox. Use an OSTimeDlyHMSM() to wait for TWO seconds.
+ */
+void readMail(void) {
+	OS_ERR err; // Make sure to check for errors and print the error code if not OS_ERR_NONE
+}
+
 /**
- * @brief Waits on the mailbox to be signaled. Once the mailbox is signaled, call readMail. Once the letter is deposited, signal the semaphore (effectively putting down the flag). Loop.
+ * @brief Waits on the mailbox to be signaled. Once the mailbox has signaled, call readMail. Loop
```

Comment:
If the new conventioon is to put the delay in the task function, why isn't it in the task here?

Reply:
It's just not an issue in this task, since it pends on the semaphore anyhow. We can put it in the task instead of the helper, but I don't think it really matters either way in this case.